server/src/attachments/dto/attachment.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { IsOptional, IsString, MaxLength, MinLength } from 'class-validator';
+
+export class AttachmentDto {
```

Comment:
Dto is only for controllers!

Reply:
Oooh, I see, thank you! Would that be an interface in this case?